### ISSUE TITLE
Remove limit for number of search results in /search/

### ISSUE THREAD

**Issue by jmcmurry** [monarch-initiative/monarch-legacy#1258]
I would like to pull this out as a separate ticket as it is extremely important.
It happened again to me yesterday looking for TGF-beta human. 

The limitation on autocomplete AND on search results means that for many of the genes that have multiple species there is literally no way to get them right now unless you go to NCBI gene and get the ID.

https://github.com/monarch-initiative/monarch-app/issues/1008#issuecomment-218265616

**Comment by mellybelly** [comment]
I searched for "shh" and could not find the mouse entry. This is a big problem!

**Comment by kshefchek** [comment]
@mellybelly regarding ssh see https://github.com/monarch-initiative/monarch-app/issues/1259, will also increase search limit.

**Comment by julesjacobsen** [comment]
@mellybelly I had the same issue with PLOD3. There were missing taxon names for Drosophilla and no mouse. Is this a wider issue? I wanted the mouse gene but wasn't able to find it.

Similarly searching for 'h3' (meaning histone H3) I was overwhelmed with human results. Would it not be better to return a 'generic' gene page using the human symbol as the identifier and then list all the orthologs there? That way the autocomplete only returns gene, disease or phenotype matches and is nicely decluttered. Genes would be the only case requiring an intermediate page.

**Comment by jmcmurry** [comment]
Agreed @julesjacobsen, it is my strong intuition that rolling up the genes to the human one would be less confusing/frustrating for the user--Both because it avoids the "can't find it" problem and because the human gene is likely to have lots of orthologs and annotations as compared to species specific genes, many of which are entirely dead ends.

I've advocated and even mocked this up on #1008; however there are two blockers.
1) Some people on the team would prefer to see some kind of higher-order taxonomic groups at a minimum)
2) Unfortunately, there were some issues determining parent gene nodes on the SciGraph back end. However, once that back end is in place, the UI would be straightforward and autocomplete likely to be substantially more snappy to boot since for genes we would have a much shorter list to look up, and no metadata concerns. It is currently extremely sluggish, often to the point of timing out. (ps. A different but related issue is returning results for diseases groups vs diseases.)

Which platforms out there in the wild are doing a great job with gene autocomplete? Can we emulate?

While rolling up to higher-order taxonomic groupings could be nice theoretically, I do worry that a) it might be time consuming to build this, b) it may not gain very much (would still require a level of redirection to a summary page) and c) it might make autocomplete even slower than it already is. My honest gut feeling? KISS. Keep it simple, species ;) However I am willing to be proven wrong if someone can point me to an autocomplete that includes all the species and does it well.

**Comment by harryhoch** [comment]
There's a definite tradeoff here. "Broader" menu structures that provide lots of choices and allow users to get to final destinations with fewer clicks are generally preferred. This would argue for having the species listed. 

The other issue in a "human" page being the default is that we'd have to make the disambiguation clear - how would users get to genes for other organisms?

Another consideration involves the goal of similarity across (roughly) analogous data types. In some sense, a parent "gene" page (even if human) collecting information across various species may not be all that far off from a disease group page (which we already have). These pages should use comparable designs to leverage any similarities.

I don't know who (if anyone) does this particularly  well. I'll try to look around.

**Comment by kshefchek** [comment]
also see https://github.com/monarch-initiative/monarch-app/pull/1260

**Comment by jmcmurry** [comment]
Going to close this ticket as the main reason for which it was opened is now fixed. Thanks so much, Kent! :)
